### DISCUSSION TITLE
skynet v1.7.0 在intel芯片的mac上编译后，启动snlua服务失败

### DISCUSSION THREAD

**Discussion by EternalSunLhx** [cloudwu/skynet#1831] [General]
macos版本：13.6.1 (22G313)  intel芯片
skynet版本：v1.7.0

正常编译，编译完启动skynet会有如下报错:
<img width="1281" alt="image" src="https://github.com/cloudwu/skynet/assets/20007843/e663d7c2-1de4-42a7-b708-848bb24d2c70">
`try open snlua failed : dlopen(./cservice/snlua.so, 0x000A): symbol not found in flat namespace '_lua_closethread'
[:00000000] Bootstrap error : snlua bootstrap`

**Comment by hanxi** [comment]
snlua.so编译的环境和skynet编译的环境是同一个lua么？

**Reply by EternalSunLhx** [reply]
确认了一下，确实是lua没有编译最新的版本

**Comment by aijit** [comment]
在 linux 下，升级编译 skynet 后，遇到同样问题。
手动 clean 一下 3rd/lua/ 目录 `rm *.o *.a *.so`  再重新编译，问题解决。
